src/main/deploy/auton/RedAmpWing1Center0.xml

```diff
@@ -2,17 +2,24 @@
 <!DOCTYPE auton SYSTEM "auton.dtd">
 <auton>
 
-<primitive id = "RESET_POSITION_PATH_PLANNER"
-            pathname = "BlueAmpWing1Center0"
-                noteOption="STATE_HOLD_FEEDER"/>
+<primitive id= "RESET_POSITION_PATH_PLANNER_NO_VISION"
```

Comment:
Why are we doing this now? We should still be using vision for the center autons. This works well. The issue for vision happens when we are at a angle.

Reply:
This is the AMP side, I thought we had switched that previously because we weren't getting good data from Photonvisio, so I'm sort of surprised that this is new.

Reply:
Ohhh sorry this was the Amp, I just saw the next one which was Center